### DISCUSSION TITLE
[FEATURE-REQUEST] Está correcto que el mensaje diga: notificaciones de escritorio cuando se usa al app mobile cotalker?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#285] [3. Ideas & Feature Requests] [answered]
Hola

Mientras sacaba SS para un manual, me di cuenta el mensaje de activar las notificaciones (imagen adjunto), y me percaté que dce escritorio, pero estoy usando la app mobile y no un desktop o laptop.

Entonces mi pregunta debería ser notificaciones a tu cuenta cotalker, en vez de escritorio?. o la frase debe ser de esa forma.

Gracias =>

![Screenshot_20210622-145732](https://user-images.githubusercontent.com/61419471/122987889-20335e00-d36f-11eb-80e2-f1b15025c7e3.png)

**Comment by NicolasPLG** [comment]
Pd: No sé como achicar el tamaño de la imagen

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG gracias por la sugerencia, efectivamente podría ser una mejora a considerar en futuros releases.
